backend/app/controllers/spree/admin/store_credits_controller.rb

```diff
@@ -74,7 +74,15 @@ def permitted_resource_params
           merge(created_by: try_spree_current_user)
       end
 
+      def user_exists?
```

Comment:
I don't think we still need this method?

Reply:
Yeah, I indeed forgot to remove it.